samples/subsys/display/lvgl/README.rst

```diff
@@ -41,6 +41,7 @@ for Arduino connectors, for example:
 - :ref:`adafruit_2_8_tft_touch_v2` and :ref:`nrf52840dk_nrf52840`
 - :ref:`buydisplay_2_8_tft_touch_arduino` and :ref:`nrf52840dk_nrf52840`
 - :ref:`ssd1306_128_shield` and :ref:`frdm_k64f`
+- :ref:`seeed_xiao_round_display` and :ref:`xiao_ble`
```

Comment:
the statement at lines 38-39 is now wrong since it's more than Arduino connectors, and now e.g. XIAO connector. Maybe rephrase as: "Display shield and a board which provides a configuration for corresponding connector, for example:"